eclair-core/src/main/scala/fr/acinq/eclair/api/Service.scala

```diff
@@ -31,6 +49,15 @@ trait Service extends Directives with Logging {
   import JsonSupport.marshaller
   import JsonSupport.formats
   import JsonSupport.serialization
+  // used to send typed messages over the websocket
+  val formatsWithTypeHint = formats.withTypeHintFieldName("type") +
+    ShortTypeHints(List(
```

Comment:
How difficult is it to provide custom mapping for type hints?

Reply:
Not difficult at all, will do

Reply:
Done in 52b17fe